doc/cli_vs_yum.rst

```diff
@@ -128,6 +128,13 @@ YUM immediately stops on a repo error, confusing and bothering the user.
 
 See the related `Fedora bug 984483 <https://bugzilla.redhat.com/show_bug.cgi?id=984483>`_.
 
+===============================================================
+ ``dnf makecache`` honors ``skip_if_unavailable`` configuration
+===============================================================
+
+The change unified behavior of DNF for all commands. DNF will not pass "dnf makecache" command if
```

Comment:
Please reword the first sentence; it should probably say "The change brings unified behavior..." or similar.

Reply:
Also, regarding the second sentence, it should rather say something like "The 'dnf makecache' command will not pass".